elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -2,8 +2,33 @@
 
 :host {
   display: block;
+  color: var(--pfe-broadcasted--color--text);
```

Comment:
Instead of `var(--pfe-broadcasted--color--text);` you can use `#{pfe-radio(color--text)};`